src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -76,24 +77,18 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 		if (tableColumns) {
 			let mycolumns: Slick.Column<any>[] = [];
 			let index: number = 0;
+
 			(<any[]>columns).map(col => {
 				if (col.type && col.type === 1) {
 					this.createCheckBoxPlugin(col, index);
 				}
 				else if (col.type && col.type === 2) {
 					this.createButtonPlugin(col);
+				} else if (col.type && col.type === 3) {
```

Comment:
You can just do `col.type === 3` - if its falsy at all this check will fail anyways.